### PR TITLE
fix templates gallery for Hydrator

### PR DESCRIPTION
**Pull request by elfenheart**

### CODE REVIEW

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by elfenheart** [comment]
On-Hold. Creating  more templates for data-pipeline

**Comment by nitinmotgi** [comment]
- Let's add the Spam / Ham pipeline. 
- Omniture Data processing pipeline
- Logs aggregation pipeline
- Omniture Data Aggregation pipeline

File: cdap-ui/templates/apps/predefined/KafkaToHBase.json [hunk 1]
```diff
@@ -1,78 +1,85 @@
...
-        "artifact": {
-          "name": "core-plugins",
-          "scope": "SYSTEM",
-          "version": "1.3.0-SNAPSHOT"
-        }
-      }
+    "artifact": {
+        "name": "cdap-etl-realtime",
+        "scope": "SYSTEM",
+        "version": "3.4.0-SNAPSHOT"
```
Comment (nitinmotgi) [inline cdap-ui/templates/apps/predefined/KafkaToHBase.json]: Might want to change to 3.4.0 instead of 3.4.0-SNAPSHOT

Comment (elfenheart) [inline cdap-ui/templates/apps/predefined/KafkaToHBase.json]: @nitinmotgi I believe this is in the release checklist

Comment (anwar6953) [inline cdap-ui/templates/apps/predefined/KafkaToHBase.json]: Yea, it is.

**Comment by ajainarayanan** [comment]
👍  LGTM

**Pull request merged**